bin/Cores/cores.json

```diff
@@ -29,7 +29,7 @@
   },
   "fbneo_libretro":{
     "name": "FinalBurn Neo",
-    "extensions": "zip",
+    "extensions": "zip|7z",
```

Comment:
This is the value returned by the core, so there's no need to explicitly specify it in the cores.json file.

I suspect it's there because it was there in fbalpha (see line 26), which is there because of this line: https://github.com/RetroAchievements/RALibretro/pull/97/files#diff-73463b17d9ee77f556b8ab8f50175adaL87

The comment suggests that the filter was there to remove `iso` and `7z`. Since `iso` is no longer there, and `7z` seems to be supported, I believe the filter can safely be removed.

Reply:
Updated to remove the extension restriction fully. And I did test locally just to confirm, you're correct on that point.